hammerblade/torch/riscv.mk

```diff
@@ -81,8 +81,6 @@ KERNEL_CSRCS   := $(notdir $(wildcard $(KERNEL_DIR)/*.c))
 KERNEL_CPPSRCS := $(notdir $(wildcard $(KERNEL_DIR)/*.cpp))
 KERNEL_OBJS    := $(patsubst %.c,%.o,$(KERNEL_CSRCS)) \
                   $(patsubst %.cpp,%.o,$(KERNEL_CPPSRCS))
-KERNEL_DIS     := $(patsubst %.c,%.dis,$(KERNEL_CSRCS)) \
-                  $(patsubst %.cpp,%.dis,$(KERNEL_CPPSRCS))
 
```

Comment:
So here is the merge conflict -- I think you reverted `fb4a35c34ba6c`

You would need to revert the revert commit ;)